src/index.html

```diff
@@ -3,10 +3,375 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo image">
+      </a>
+      <nav>
+        <ul class="headerList">
+          <li class="headerList_link">
+            <a
+              class="is-active" href="#apple">APPLE</a>
+          </li>
```

Comment:
try to use one case for naming classes. Check other cases

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              class="nav__link is-active" href="#apple">APPLE</a>
          </li>
```